getting-started-2.0.md

```diff
@@ -0,0 +1,155 @@
+1. Type `cd <my_repo>`
+1. Type `hlx up &`
+   <br>This will build a _Helix Pages_ project locally, start a local web server in the background and open a new browser window at [`http://localhost:3000/`](http://localhost:3000/ "your local Helix server") when done.
+1. View your page rendered locally by _Helix Pages_ using the default page styling.
+1. Type `code .` to open this project in the IDE
+1. Create a `style.css` file and insert your CSS rules. See the [default CSS](https://raw.githubusercontent.com/adobe/helix-pages/master/htdocs/style.css "Default CSS") for pointers where you can add overrides.
+1. Save your changes.
+1. Reload [`http://localhost:3000/`](http://localhost:3000/ "your local Helix server") to preview your style changes locally.
+1. Repeat steps 8 - 10 until you are happy with how your page looks.
+1. Type `git add style.css && git commit -m"custom styling" && git push`
+   <br>This will persist your local style changes to the GitHub repository.
+1. Now reload `https://<my_repo>-<my_org>.project-helix.page/` and admire your published page with custom styling. 
+
+> **Hint:** If you use `hlx up` with the `--no-open` argument, it won't open a browser window.
+
+## Add your own code
+
+> You'll need:
+> - basic JavaScript knowledge
+> - basic HTML knowledge
+
+_Helix Pages_ allows you to control the rendering of your content by changing the HTL template and adding custom JavaScript code.
+
+1. Open your favorite shell and change to the directory where you checked out your GitHub repository, e.g. type `cd ~/code/<my-repo>`.
+2. Type `code .` to open this project in the IDE.
+3. Create a `src` folder.
+4. Add an `html.pre.js` file in there. For example, start with the [default one](https://raw.githubusercontent.com/adobe/helix-pages/master/src/html.pre.js "default html.pre.js").
+5. Inside the `pre` function, add something to the `content` object, e.g.
+```js
+context.content.time = `${new Date()}`;
+```
+6. Save your changes.
+7. Add an `html.htl` in the same folder. Again, you can inspire from [the default](https://raw.githubusercontent.com/adobe/helix-pages/master/src/html.htl "default html.htl").
+8. Display your custom addition., e.g. right after the `<body>` tag, add:
+```html
+<div>
+  <em>Helix Pages generated this page on ${content.time}</em>
+</div>
+```
+9. Save the changes.
+10. Reload [`http://localhost:3000/`](http://localhost:3000/ "your local Helix server") to see the page rendered locally using your custom code.
+11. Type `git add src/* && git commit -m"custom code" && git push`
+    <br>This will commit your local code changes to the GitHub repository.
+12. Go to `https://<my-repo>-<my_org>.project-helix.page/` and observe the published page being rendered with your custom code. 
+
+
+> **Note:** You could also add custom code directly in GitHub, but we recommend against writing code outside a proper development environment.
```

Comment:
why is this note neccesairy?

Reply:
I wanted to point out that having a local dev env isn't strictly mandatory to work with Helix Pages. Would you rephrase it or remove it?